lib/networkCall.js

```diff
@@ -0,0 +1,17 @@
+const fetch = require('node-fetch')
+
+const networkCall = async (url, method = 'get', body = null, headers = null) => {
```

Comment:
What is the benefit of this function?
Does directly using `fetch` in our service functions lack anything?
Is this done to stringify the response?

Reply:
Mentioned the purpose in issue description
https://github.com/Real-Dev-Squad/website-backend/issues/84

```
Make a wrapper function for network calls so that whenever we want to perform a network call, we can call this function.
In case we choose to use another library for network calls we can just make changes in this particular function rather than changing the code at every network call. This makes our code modular.
```